studio/src/aem/mas-filter-toolbar.js

```diff
@@ -29,6 +30,11 @@ class MasFilterToolbar extends LitElement {
         this.variant = 'all';
     }
 
+    connectedCallback() {
+        super.connectedCallback();
+        this.searchText = document.querySelector('mas-studio').searchText;
```

Comment:
cannot this get the searchText value from deeplink?
also, I propose to drop the `mas-` prefix in the file names. It is already the MAS repository.

Reply:
i could, but this also triggers a event handler creation i'm not a fan of here